packaging/helm-charts/helm3/strimzi-kafka-operator/values.yaml

```diff
@@ -72,6 +75,7 @@ podDisruptionBudget:
   # If both values are set, the template will use the first one and ignore the second one. currently by default minAvailable is set to 1
   minAvailable: 1
   maxUnavailable:
+  unhealthyPodEvictionPolicy: AlwaysAllow
```

Comment:
Could you please add the new field also to https://github.com/strimzi/strimzi-kafka-operator/blob/main/packaging/helm-charts/helm3/strimzi-kafka-operator/README.md?

Reply:
Added